sql/moz-fx-data-shared-prod/pocket/spoc_tile_ids/metadata.yaml

```diff
@@ -0,0 +1,5 @@
+friendly_name: Sponsored Content (Spoc) Tile IDs
+description: |-
+  Lookup table of the currently registered tile IDs for Pocket sponsored content
```

Comment:
@kirill-demtchouk I would appreciate getting your eyes on the `metadata.yaml` files and `schema.yaml` files here to see if you disagree with any of the documentation or field descriptions.